contracts/library/MoolaStakingRewards.sol

```diff
@@ -0,0 +1,35 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.4;
+
+import "hardhat/console.sol";
+
+import "../openzeppelin-solidity/contracts/SafeERC20.sol";
+import "../ubeswap-farming/interfaces/IMoolaStakingRewards.sol";
+
+/**
+ * MoolaStakingRewards is a library containing helper functions for interacting with contracts
+ * implementing the IMoolaStakingRewards interface.
+ **/
+library MoolaStakingRewards {
```

Comment:
i can't imagine another farm bot that will need this, and the functions are pretty trivial. Not sure it's useful to live separately from the ubeswap farm bot contract